Backend/Tournament_Public/server.js

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
+const express = require('express');
+const publicUserRoutes = require('./routes/publicUserRoutes');
+
```

Comment:
Think might need to add CORS for frontend calling here

Reply:
When you do this try make the ORIGIN an env var

Reply:
ok  boss. my CORS_ORIGIN=http://localhost:3000 is liddis right in env file

Reply:
shld be the frontend addr ah so the port depends on the frontend but can change ltr anyways so not a big deal